data/patches/miq_application.js.diff

```diff
@@ -1,37 +1,52 @@
---- miq_application.js	2016-05-09 15:44:20.785132210 -0400
-+++ miq_application.js.new	2016-05-10 02:12:20.945132210 -0400
-@@ -1,5 +1,37 @@
+--- miq_application.js	2016-05-23 15:46:51.571062620 +0200
++++ miq_application.js.new	2016-05-23 15:48:51.438832567 +0200
+@@ -1,5 +1,52 @@
  // MIQ specific JS functions
 
 +// MiqQE section
 +// ---------------
 +
 +// Initialize MiqQE
-+var MiqQE = {};
-+
-+function setMiqQE(key, value) {
-+  MiqQE[key] = value;
-+}
-+
-+function getMiqQE(key) {
-+  // Returns undefined if key is undefined, otherwise value of key
-+  return MiqQE[key];
-+}
++// Make sure to update version here and in pytest_selenium when changed
++var MiqQE = {'autofocus': 0, 'debounce': 0};
++var MiqQE_version = 1;
```

Comment:
I don't like this, because we currently have to bump the version here and then bump the version check in pytest_selenium as well
The alternative would be to either store it in js.py, replace the value during patching in this diff and pull it from js.py to pytest_selenium or we could parse it directly from this diff in pytest_selenium.
But this works now.